src/toolkit/content/widgets/tabbox-js.patch

```diff
@@ -2,7 +2,15 @@ diff --git a/toolkit/content/widgets/tabbox.js b/toolkit/content/widgets/tabbox.
 index 70afbfc87d543971e6f8a0661a44b682920a7bc4..2f767296db8043318fab2aeb39bfc5eee090b258 100644
 --- a/toolkit/content/widgets/tabbox.js
 +++ b/toolkit/content/widgets/tabbox.js
-@@ -213,7 +213,7 @@
+@@ -125,6 +125,7 @@
+
+       const { ShortcutUtils } = imports;
+
++      return;
```

Comment:
I've thought about it, instead of adding a return here (which would invalidate the use of cmd+opt+arrows), instead I would remove just this line:

https://searchfox.org/firefox-main/source/toolkit/modules/ShortcutUtils.sys.mjs#349